jobs/J_NCEPPOST

```diff
@@ -27,7 +27,7 @@ export KEEPDATA=${KEEPDATA:-NO}
 ####################################
 # File To Log Msgs
 ####################################
-export jlogfile=${jlogfile:-${COMROOT}/logs/jlogfiles/jlogfile.${job}.${pid}}
+#export jlogfile=${jlogfile:-${COMROOT}/logs/jlogfiles/jlogfile.${job}.${pid}}
```

Comment:
Perhaps just remove this line now that we're retiring the use of jlogfiles?

Reply:
@KateFriedman-NOAA done!